README.md

```diff
@@ -1,24 +1,183 @@
-# README
+<a name="readme-top"></a>
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+<!-- TABLE OF CONTENTS -->
 
-Things you may want to cover:
+# 📗 Table of Contents
```

Comment:
- [ ]  You are using rswag tests, but the "test" folder still exists, which is used by another test framework. Kindly consider deleting that folder if you are not using it. 👍